capstone-app/src/app/modules/photos/components/menu-item/menu-item.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, Input, ViewChild, Output, EventEmitter } from '@angular/core';
+import { NavItem } from '../../nav-item';
+import { MatMenu } from '@angular/material/menu';
+
+@Component({
+  selector: 'app-menu-item',
+  templateUrl: './menu-item.component.html',
+  styleUrls: ['./menu-item.component.scss']
+})
+export class MenuItemComponent {
+  markerFilter: string;
```

Comment:
I'd recommend putting the decorator properties first (the ones with Input, Output, etc). And I think this can be set to private.